__fixtures__/example-todo-main-with-errors/web/src/Routes.js

```diff
@@ -15,6 +15,7 @@ const Routes = () => {
       <Route path="/" page={HomePage} name="home" />
       <Route path="/" page={HomePage} name="home" />
       <Route path="/{foo}/{foo}" page={DoesNotExist} name="something-else" />
+      {/* <Route path="/{bar}/{bar}" page={DoesNotExist} name="something-different" /> */}
```

Comment:
I know this is added for one of the tests. But removing it doesn't make any test fail, right?
So how about we just add another comment, saying this is here for test purposes? Otherwise I think someone might just find it in the future and remove it in an effort to "clean up".
```suggestion
      {/* The comment below is for testing purposes, please don't remove */}
      {/* <Route path="/{bar}/{bar}" page={DoesNotExist} name="something-different" /> */}
```

Reply:
Yeah this was originally added just to make sure my regex logic handled comments. It's no longer really needed, I'm sure that structure's ast doesn't pick up the commented out route, but if you want to keep it in then we add this. Up to you if you want to just remove it instead?

Reply:
If you're good with removing it, so am I! 

```suggestion
```